jobs/socorro_import_crash_data.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import logging
+import tempfile
+import urllib2
+
+from datetime import datetime as dt, timedelta, date
+from os import environ
+
+from pyspark.context import SparkContext
+from pyspark.sql.session import SparkSession
+from pyspark.sql.types import *
+
+"""
+Adapted from:
+https://github.com/mozilla-services/data-pipeline/blob/0c94d328f243338d21bae360547c300ac1b82b12/reports/socorro_import/ImportCrashData.ipynb
```

Comment:
I'm not familiar with this file (though apparently I'm the last person to have edited it heh) but I'm going to assume this is a relatively straight-forward adaption of existing working code, and that this adaption has been tested.

Reply:
Yup it has been tested.